src/main.ts

```diff
@@ -227,4 +229,53 @@ function defaultLabelCheck(){
       core.setFailed('Please set default label as need-to-triage')
   })
 }
+
+async function standardLabelsCheck(repository: string, ownername: string, secret_token: string, octokit: Octokit){
+  try{
+    const result = await octokit.request('GET /repos/{owner}/{repo}/labels',{
+      repo: repository,
+      owner: ownername,
+      headers : { 
+        Authorization: 'Bearer ' + secret_token,
+      },
+    }); 
+    let labelArray = result.data;
+    var map = new Map();
```

Comment:
Why are we using a Map?

Reply:
To avoid O(n^2) complexity

Reply:
Changed 1 to true

Reply:
Instead of map, you can use Set. Explore that option.

Reply:
Also, use variable names which are more intuitive. So instead of "map", something like existingLabels will be more suitable

Reply:
Done!